OpenCL_C.txt

```diff
@@ -307,7 +307,8 @@ The following table describes the list of built-in scalar data types.
       The `double` data type must conform to the IEEE 754 double precision
       storage format.
 
-      <<unified-spec, Requires>> support for OpenCL C 1.2 or newer.
+      <<unified-spec, Requires>> support for OpenCL C 1.2 or newer. In
```

Comment:
Should this change be duplicated for the double vector types?

Reply:
Done! Thanks!